### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by xerta555**

### CODE REVIEW

**Review by yujincheng08 — APPROVED** [review]

**Comment by xerta555** [comment]
Wich line(s) block the check?

**Comment by Alexey71** [comment]
> 
> 
> Wich line(s) block the check?

https://github.com/LSPosed/LSPosed/runs/3351641660?check_suite_focus=true#step:7:121

**Comment by xerta555** [comment]
The line number 88 must be removed

**Comment by xerta555** [comment]
I hope it will good now...Github is really strange for some taks.

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -69,17 +72,19 @@
...
+    <string name="verbose_log_not_avaliable">Le journal verbeux n’est pas disponible, si vous l\’avez activé sans avoir redémarré, essayez à nouveau après avoir redémarrer.</string>
 
     <!-- Notification -->
     <string name="module_is_not_activated_yet">Le module LSPosed n\’est pas encore actif</string>
     <string name="module_is_not_activated_yet_detailed">%s à été installé, mais il n\’est pas encore installé</string>
-    <string name="xposed_module_updated_notification_content">%s à été mis à jour</string>
+    <string name="xposed_module_updated_notification_title">Module Xposed mis à jour</string>
+    <string name="xposed_module_updated_notification_content">%s à été mis à jour, merci de forcer l\’arrêt ou de redémarrer les applis dans leurs champs d\’application</string>
+    <string name="xposed_module_updated_notification_title_system">Module Xposed à jour, redémarrage du système requis</string>
+    <string name="xposed_module_updated_notification_content_system">%s à été mis à jour, puisque le champ d'application contient le Sous Système, le redémarrage est nécessaire pour appliquer les changements</string>
```
Comment (yujincheng08) [inline app/src/main/res/values-fr/strings.xml:83]: \\'

**Review by yujincheng08 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (xerta555) [reply app/src/main/res/values-fr/strings.xml:83]: I think i just fix it now in my last update on my repo

**Comment by xerta555** [comment]
Should be fixed definitivly now...

**Review by yujincheng08 — APPROVED** [review]

**Pull request merged**